plugins/woocommerce-blocks/assets/js/atomic/blocks/product-elements/stock-indicator/block.tsx

```diff
@@ -10,13 +10,15 @@ import {
 import { useStyleProps } from '@woocommerce/base-hooks';
 import { withProductDataContext } from '@woocommerce/shared-hocs';
 import type { HTMLAttributes } from 'react';
+import { ProductResponseItem } from '@woocommerce/types';
 
 /**
  * Internal dependencies
  */
 import './style.scss';
 import type { BlockAttributes } from './types';
 
+const ALLOWED_PRODUCT_TYPES = [ 'simple', 'variation' ];
```

Comment:
I think it should be extensible

Reply:
Good call! I made this extensible in the commit [fd74b54](https://github.com/woocommerce/woocommerce/pull/53319/commits/fd74b542854ec6b4135992f76437fdd1312d7df1)

Reply:
Ups, sorry, I left a comment about this before seeing this thread. Copying it here instead:

> I wonder if we should mark this filter as experimental, at least for now. Or maybe even remove it until we find a use case where it's needed. (I'm sure there are, but I'm a bit wary of adding the filter so early in the development cycle when we hadn't started looking into extensibility yet). What do you think?

Reply:
Good idea! Let's remove the filter for now and add it back later in the dev process.

Reply:
Done in the commit [180da0a](https://github.com/woocommerce/woocommerce/pull/53319/commits/180da0a13e59948c5edb004f452f5cac846e35da)